web/packages/api/src/environment.ts

```diff
@@ -341,9 +377,35 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
                         id: "WETH",
                         address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
                         minimumTransferAmount: 15_000_000_000_000n,
-                    }
+                    },
+                    {
+                        id: "PILT",
+                        address: "0x99E743964C036bc28931Fb564817db428Aa7f752",
+                        minimumTransferAmount: 1n,
+                    },
                 ],
-            }
+            },
+            {
+                id: "PILT",
+                name: "PILT",
+                type: "substrate",
+                destinationIds: ["assethub"],
+                paraInfo: {
+                  paraId: 2086,
+                  destinationFeeDOT: 0n,
+                  skipExistentialDepositCheck: false,
+                  addressType: "32byte",
+                  decimals: 15,
+                  maxConsumers: 16,
+                },
+                erc20tokensReceivable: [
+                  {
+                    id: "PILT",
+                    address: "0x99E743964C036bc28931Fb564817db428Aa7f752",
```

Comment:
What would happen if someone tries to directly send this token to PILT from ethereum not using polar path?

Reply:
Also maybe this should live outside the context/environment config? As it is only used outside the API package.